### PR TITLE
add `cupy.random.zipf`

### PR DESCRIPTION
**Pull request by YoshikawaMasashi**

### CODE REVIEW

**Comment by YoshikawaMasashi** [comment]
Jenkins, test this please.

File: cupy/random/distributions.py [hunk 1]
```diff
@@ -418,3 +418,29 @@ def vonmises(mu, kappa, size=None, dtype=float):
...
+
+
+def zipf(a, size=None, dtype=int):
+    """Zipf distribution.
+
+    Returns an array of samples drawn from the Zipf distribution. Its
+    probability mass function is defined as
+
+    .. math::
+        f(x) = \\frac{x^{-a}}{ \\zeta (a)},
```
Comment (toslunar) [inline cupy/random/distributions.py]: It would be mentioned that \zeta is the Riemann zeta function.

File: cupy/random/generator.py [hunk 2]
```diff
@@ -453,6 +453,21 @@ def uniform(self, low=0.0, high=1.0, size=None, dtype=float):
             high = cupy.asarray(high, dtype)
         return RandomState._scale_kernel(low, high, rand)
 
+    def zipf(self, a, size=None, dtype=int):
+        """Returns an array of samples drawn from the Zipf distribution.
+
+        .. seealso::
+            :func:`cupy.random.zipf` for full documentation,
+            :meth:`numpy.random.RandomState.zipf`
+        """
```
Comment (toslunar) [inline cupy/random/generator.py]: Please check `a > 0`.

**Comment by YoshikawaMasashi** [comment]
Jenkins, test this please.

**Pull request merged**